modules/mediagoBidAdapter.js

```diff
@@ -415,6 +428,26 @@ export const spec = {
     return bidResponses;
   },
 
+  getUserSyncs: function (syncOptions, serverResponse, gdprConsent, uspConsent, gppConsent) {
+    const origin = encodeURIComponent(location.origin || `https://${location.host}`);
+    const url = `${COOKY_SYNC_URL}?dm=${origin}&gdpr_consent=${gdprConsent.consentString}`;
```

Comment:
When running this in simple testing, I'm getting this error.

mediagoBidAdapter.js:433  Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'consentString')
    at Object.getUserSyncs (mediagoBidAdapter.js:433:76)
    at bidderFactory.js:512:22
    at registerSyncs (bidderFactory.js:347:5)
    at bidderFactory.js:265:11
    at Object.runWithBidder (config.js:508:14)
    at afterAllResponses (bidderFactory.js:263:9)
    at perfMetrics.js:288:17
    at utils.js:661:12
    at bidderFactory.js:448:7
    at runWithBidder (config.js:508:14)
    

I'm not setting up any of the gdpr things for this test, just a simple test in hello_world.html. I would assume you want this to function to work in this situation as well. I think you should be checking if these variables are set in this function. You can see the various ways people are checking for this in other adapters.